### ISSUE TITLE
MMM-Remote-Control broken after MM2 upgrade to 2.11.0

### ISSUE THREAD

**Issue by Moskito99** [Jopyth/MMM-Remote-Control#185]
Hi!
This module is not working anymore with MagicMirror2 2.11.0

Most of the menus show undefined.
When click the entrys nothing happens.
The menu updeates as example neve show any modul.

Even a complete reinstall is not working.

**Comment by yurMoThEr** [comment]
OMG, I thought I was the only one!

**Comment by mattx38** [comment]
Oh no, you're not alone :-/

**Comment by michael-oetjen** [comment]
Hi there, I noticed, that the socket URL is wrong:
instead of
"/remote.htmlsocket.io/?EIO=3&transport=polling&t=N4_yb4O" it should say
"/remote.html?socket.io/?EIO=3&transport=polling&t=N4_yb4O"

Notice the question mark between "html" and "socket.io"

I don't know where to change that, though ;-(

**Comment by khassel** [comment]
never wrote something about extra slashes.

replace
````
	// Private Methods
	self.socket = io("/" + self.moduleName, {
		path: window.location.pathname + "socket.io"
	});
````
with
````
	// Private Methods
        self.socket = io("/" + self.moduleName);
````

in `js/socketclient.js`

**Comment by RoadRunnr1469** [comment]
Excellent. it is working. Thanks

**Comment by alomar-mac** [comment]
WOW it is working
Thank you so much





> never wrote something about extra slashes.
> 
> replace
> 
> ```
> 	// Private Methods
> 	self.socket = io("/" + self.moduleName, {
> 		path: window.location.pathname + "socket.io"
> 	});
> ```
> 
> with
> 
> ```
> 	// Private Methods
>         self.socket = io("/" + self.moduleName);
> ```
> 
> in `js/socketclient.js`

WOW it is working
Thank you so much

**Comment by ezar** [comment]
Same problem here!

**Comment by chen32g** [comment]
great, solve for me with new Magic version

**Comment by Maype2988** [comment]
Perfect!!! Working for too, thanks you

**Comment by rockman55** [comment]
I have been looking for a solution on this problems for days. Thanks so much for the fix.

**Comment by nickosouras** [comment]
I cant find the file that needs to be changed? maybe im just dumb.

**Comment by RoadRunnr1469** [comment]
inside the MagicMirror folder is a folder js
the file you need to edit is socketclient.js
by doing a SSH it looks something like this
**username**@**magicmirrornsystem**:~/MagicMirro/js/ $

**Comment by nickosouras** [comment]
thank you!

**Comment by alyssacaples** [comment]
I tried fixing that issue in the socketclient.js with khassel's solution and i am still unable to access any of the features

**Comment by marco97233** [comment]
Merci a vous tous cela fonctionne pour moi maintenant!!!!

**Comment by cain0z** [comment]
Thank you it is now working as I expected.

**Comment by dvbit** [comment]
uhm I pathched the file but nevertheless the modules are not controlled 
I have no more "loading..."
but I cannot hide or display modules.
Does it work for you?

PS: this is a brand new install.

**Comment by eskiiom** [comment]
thanks, that works perfectly for me too

**Comment by ayed97** [comment]
You are a hero 🥇

**Comment by ezeholz** [comment]
Hey all! After following this issue, seems to be resolved in version 2.12+ of MagicMirror. If you're having the same error, just try updating both MagicMirror and Remote Control, and try again. Keep programming people! ;D
